grammar.js

```diff
@@ -468,7 +468,7 @@ module.exports = grammar({
         '=',
         repeat($.decorator),
         $.expression,
-        repeat(alias($._let_binding_and, $.let_binding)),
+        optional(alias($._let_binding_and, $.let_binding)),
```

Comment:
I have no clear idea why it is generating an error (#171).

My hypothesis: As the rule `_let_binding_and` has a `_let_binding` rule so it's already _recursive_.

Reply:
Perhaps, yes. I also see no precedence specs for `let_binding`, `_let_binding`, `_let_binding_and`. Maybe that’s what involved too.

Nevertheless, I think the new variant is fine too.